### ISSUE TITLE
[Error Audit] Category 3 error messages clean-up

### ISSUE THREAD

**Issue by ADPennington** [raft-tech/TANF-app#2792]
**Description:**

This ticket will track the requirements for cleaning up error message language for category 3 type errors (i.e. Record value consistency)

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_
- [x] All validator f strings should have Item Numbers / Friendly Names in both the error message and the relevant item number / name columns
- [x] Technical "validator1" type language is removed from error messages
- [x] If there are any straggler errors that list a greater than or equal to and smaller than or equal to to define the bounds, we should list a simple range or a set of values instead
- [ ] Parser outputs are consistent with desired outputs listed in the cat 3 table of the [error audit](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/lIHaHcbpsiyAKVhaisxx/#cat-3-validators)
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [x] Implement item numbers / friendly names
- [ ] Check for consistency with AC & [audit table](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/lIHaHcbpsiyAKVhaisxx/#cat-3-validators)
- [ ] Run Testing Checklist and confirm all tests pass

**Supporting Documentation:**
_Please include any relevant log snippets/files/screen shots_

- [Error Audit](https://tdp-project.gitbook.io/project-trackers-and-guides#cat-3-validators)

**Comment by jtimpe** [comment]
Depends on #2693

**Comment by ADPennington** [comment]
@reitermb @robgendron AC # 2 in this ticket needs to be prioritized for parity.  this AC is focused on fixing incorrect error messaging, while the error audit-related tickets are more focused on aligning the error messages with the style guide. 

I converted it into a separate ticket: #2916

**Comment by ADPennington** [comment]
> @reitermb @robgendron AC # 2 in this ticket needs to be prioritized for parity. this AC is focused on fixing incorrect error messaging, while the error audit-related tickets are more focused on aligning the error messages with the style guide.
> 
> I converted it into a separate ticket: #2916

@reitermb @robgendron @elipe17 looks like this actually has already been fixed. so i closed #2916

**Comment by robgendron** [comment]
Will build out next UX Sync, templated to be worked on Sprint 100.

**Comment by robgendron** [comment]
Will refine next week/potential for following sprint.

**Comment by ADPennington** [comment]
@reitermb the item numbers for cat 3 errors are not currently included in the error report. if you agree that they should be, please add it as an AC to this ticket. 

![2962p2](https://github.com/raft-tech/TANF-app/assets/63075587/679320b7-83a9-433e-817a-3e979e91a1b3)

**Comment by robgendron** [comment]
Update: All validator f strings should have Item Numbers / Friendly Names in both the error message and the relevant item number / name columns - completed

**Comment by robgendron** [comment]
Still researching - will follow up with @jtimpe.

**Comment by reitermb** [comment]
3 pts remaining

**Comment by elipe17** [comment]
PR going up today, 2 points remaining.
